encrypted-media-respec.html

```diff
@@ -657,9 +669,10 @@
 
             <ol class="method-algorithm">
               <!-- TODO: Convert all parameters to use <code>. -->
+              <li><p>Let <var>document</var> be the calling context's <a def-id="document-concept"></a>.</p></li>
+              <li><p>If <var>document</var> is not <a data-cite="!HTML#allowed-to-use">allowed to use</a> the <a data-cite="!FEATURE-POLICY#policy-controlled-feature">policy-controlled feature</a> named <a>encrypted-media</a>, return a promise rejected with a newly created <a def-id="NotSupportedError"></a>.</p></li>
```

Comment:
This could be a SecurityError as recommended in https://github.com/WICG/feature-policy/blob/gh-pages/integration.md. I actually think SecurityError is more appropriate but there may be backward compatibility concerns.

Reply:
I think that's probably fine. NotSupportedError is an expected result that needs to be handled, but nothing is going to work in this case anyway.

@xhwang-chromium, we'll need to update Chromium if this changes (along with the actual algorithm steps).

Reply:
Sorry just to clarify: which do you prefer - NotSupportedError or SecurityError?

Reply:
I am fine with `SecurityError` if that is what other specs are using.

Reply:
https://github.com/w3c/web-platform-tests/pull/7889 assumes SecurityError.

Reply:
Updated link: https://github.com/w3c/webappsec-feature-policy/blob/master/integration.md#referencing-the-feature-policy-spec